modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -64,9 +63,13 @@ Returns the requested number of approximate nearest neighbor nodes and their sim
 | `SHOW INDEXES WHERE type = "VECTOR"`
 | Lists all vector indexes, see the xref:indexes-for-search-performance.adoc#indexes-list-indexes[`SHOW INDEXES`] command for details. There is no vector index filter built into `SHOW INDEXES`.
 
+| Set vector property.
+| {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`]
+| label:beta[] Update a given node property with the given vector in a more space-efficient way than directly using xref:clauses/set.adoc#set-set-a-property[`SET`].
+
 | Set vector property.
 | {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`]
-| Update a given node property with the given vector in a more space-efficient way than directly using xref:clauses/set.adoc#set-set-a-property[`SET`].
+| label:beta[] label:deprecated[] Please use {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`]
```

Comment:
We try to avoid using "please" in the docs.
```suggestion
| label:beta[] label:deprecated[] It is replaced by {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`]
```